reporter/http/http.go

```diff
@@ -37,10 +38,15 @@ const (
 	defaultMaxBacklog    = 1000
 )
 
+// Doer will do a request to the Zipkin HTTP Collector
+type Doer interface {
```

Comment:
Interestingly Prometheus go lib also has this interface but there it is called `HTTPDoer`. Maybe for sake of readability and familiarity for quite a few folks we might want to call it that too.

See: https://github.com/prometheus/client_golang/blob/master/prometheus/push/push.go#L60

Reply:
Nice finding. My only concern is that the package name is also `http` so it will end up being `http.HTTPDoer`.

Reply:
Not too worried about it as there is no reason to point to this interface anywhere outside of this package.